src/main/java/org/spongepowered/api/service/context/Context.java

```diff
@@ -31,11 +31,12 @@
 import java.util.Map;
 
 /**
- * The context that a permission check occurs in. Instances of a context are
+ * The context that a service check occurs in. Instances of a context are
  * designed to function as cache keys, meaning they should be fairly lightweight
  * and not hold references to large objects
  */
 public final class Context implements Map.Entry<String, String> {
+    public static final String USER_KEY = "user";
```

Comment:
What do you plan on using this user context key for? Accounts already are exposed by player -- would this be used in addition?

From my work with contexts in the past, having a context for a user type is probably a road to pain and suffering, but maybe I'm not getting what you're trying to do here.